.travis.yml

```diff
@@ -19,6 +19,13 @@ sudo: required
 services:
   - docker
 
+install:
+  - . $HOME/.nvm/nvm.sh
+  - nvm install 8.9.0
+  - nvm use 8.9.0
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+
 cache:
   directories:
-  - $HOME/.m2
\ No newline at end of file
+  - $HOME/.m2
+  - modules/web/node_modules
```

Comment:
Should this be "composer/modules/web/" ?

Reply:
Yes, it should be now. I'll update it.